Ihor/Netflix/src/redux/Sagas/FetchSaga.ts

```diff
@@ -35,35 +26,67 @@ export const fetchSelectedMovie = async (id: string): Promise<IMovie> => {
   return movie;
 };
 
-function* requestMoviesSaga(action: IMoviesRequested) {
+
+interface IMoviesRequested{
```

Comment:
don't like that you had to create an interface just to use it in saga.  Seems redundant to me. My proposal is to use this instead:
`ReturnType<typeof  moviesRequested >
`